packages/import-maps-generate/README.md

```diff
@@ -0,0 +1,104 @@
+# Create Import Map
+
+[//]: # (AUTO INSERT HEADER PREPUBLISH)
+
+This will allow you to generate an [import-map](https://github.com/WICG/import-maps).
+
+::: warning
+Only yarn.lock is supported for now
+:::
+
+## Features
+- generates a flat import-map
+- prompts when there are conflicts
+- supports resolutions, overrides, deletions via a config property in package.json
+- supports yarn workspaces (monorepos)
+
+## Usage
+
+Install:
+
+```bash
+yarn add @import-maps/generate
+```
+
+Generation of the importMap should happen after you install dependencies, so you can add the script to your postinstall hook in `package.json`:
+
+```json
+"scripts": {
+  "postinstall": "generate-import-map"
+}
+```
+
+If you only want to try it out once to see what it will generate you can simply do so via:
+
+```bash
+npx @import-map/generate
+```
+
+## Configuration
+
+There may be times where you'll want to apply overrides. For example, if you're using the built-in `std:kv-storage` module and the polyfill for browsers that don't support it, the `kv-storage-polyfill` will be in your yarn.lock, and will be included in your import map. But what you'll actually want is:
+
```

Comment:
It is not clear here how to solve this problem

Reply:
jup still working on this 🙈

Reply:
The JSON example down here shows how to solve it right?

Reply:
explained it more in detail hope it's more clear now